docs/UserGuide.adoc

```diff
@@ -130,7 +130,7 @@ Format: `whitelist`
 === Editing a person : `edit`
 
 Edits an existing debtor, blacklisted/whitelisted contact in the address book. +
-Format: `edit [INDEX] [n/NAME] [hp/HANDPHONE_NUMBER] [home/HOME_PHONE_NUMBER] [op/OFFICE_PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [pc/POSTAL_CODE] [d/DEBT] [dl/DEADLINE] [int/INTEREST] [t/TAG]...`
+Format: `edit [INDEX] [n/NAME] [hp/HANDPHONE_NUMBER] [home/HOME_PHONE_NUMBER] [op/OFFICE_PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [pc/POSTAL_CODE] [d/DEBT] [td/TOTAL DEBT] [dl/DEADLINE] [int/INTEREST] [t/TAG]...`
```

Comment:
update the command summary part near the bottom of the userguide as well

Reply:
thank you!